### PR TITLE
Branchv1.2 testing optional fields

### LINKED ISSUE #95: Change error massage for opening hours error
image

### LINKED ISSUE #96: Create test cases to test for optional fields
The test case does not test for optional fields such as opening hours, rating, price range and description

### PR DESCRIPTION
**Pull request by peter-yeh**

### CODE REVIEW

**Comment by peter-yeh** [comment]
Merge this after Add itinerary #98 has merged

**Review by yuxuanxc — APPROVED** [review]

**Pull request merged**
